src/tests/dictionaryConcepts/container/DictionaryConcepts.test.jsx

```diff
@@ -430,4 +440,86 @@ describe('Test suite for dictionary concepts components', () => {
     expect(mapStateToProps(initialState).loading).toEqual(false);
     expect(mapStateToProps(initialState).dictionaries).toEqual([]);
   });
+
+  it('should update the state with search term on change in the search bar', () => {
+    const props = {
+      match: {
+        params: {
+          typeName: 'dev-col',
+          type: 'orgs',
+          collectionName: 'dev-col',
+          dictionaryName: 'dev-col',
+        },
+      },
+      location: {
+        pathname: '/random/path',
+      },
+      fetchDictionaryConcepts: jest.fn(),
+      concepts: [],
+      filteredClass: ['Diagnosis'],
+      filteredSources: ['CIEL'],
```

Comment:
Don't we already have a constant for this?

Reply:
@dkayiwa no we don't and I didn't want to refactor the test file in this PR.

Reply:
@dkayiwa I have made the changes...